frontend/packages/dev-console/locales/zh/devconsole.json

```diff
@@ -0,0 +1,314 @@
+  "Builder Image": "构建器镜像",
+  "Detecting recommended builder images...": "检测推荐的构建器镜像...",
+  "Builder image(s) detected.": "检测到的构建器镜像。",
+  "Recommended builder images are represented by": "推荐的构建器镜像由表示",
+  "icon": "图标",
+  "Unable to detect the builder image.": "无法检测构建器镜像。",
+  "Select the most appropriate one from the list to continue.": "从列表中选择最合适的一个来继续。",
+  "Builder Image Version": "构建器镜像版本",
+  "Builder": "构建器",
+  "Sample repository:": "样本仓库：",
+  "Create": "创建",
+  "Deploy Image": "部署镜像",
+  "Hide Advanced Git Options": "隐藏高级 Git 选项",
+  "Show Advanced Git Options": "显示高级 Git 选项",
+  "Git Reference": "Git 参考",
+  "Optional branch, tag, or commit.": "可选 branch、tag 或 commit。",
+  "Context Dir": "Context Dir",
+  "Optional subdirectory for the application source code, used as a context directory for build.": "可选的应用程序源代码的子目录，用作构建的上下文目录。",
+  "Dockerfile": "Docker",
+  "Dockerfile Path": "Dockerfile 路径",
+  "Allows the builds to use a different path to locate your Dockerfile, relative to the Context Dir field.": "允许构建使用不同的路径来定位您的 Dockerfile，相对于 Context Dir 字段。",
+  "Container Port": "容器端口",
+  "Port number the container exposes.": "容器公开的端口号。",
+  "Validating": "验证",
+  "Validated": "已验证",
+  "URL is valid but cannot be reached. If this is a private repository, enter a source secret in Advanced Git Options": "URL 有效但无法访问。如果这是一个私有仓库，在高级 Git 选项中输入源 secret",
+  "Git": "Git",
+  "Git Repo URL": "Git Repo URL",
+  "Git Type": "Git 类型",
+  "Defaulting Git Type to Other": "其他的默认 Git 类型 ",
+  "We failed to detect the Git type.": "无法检测到 Git 类型。",
+  "Try Sample": "尝试样本",
+  "Source Secret": "源 Secret",
+  "Secret with credentials for pulling your source code.": "用于拉取源代码的带有凭证的 Secret。",
+  "Create New Secret": "创建新 Secret",
+  "Enter an image name": "输入镜像名称",
+  "To deploy an image from a private repository, you must": "要从私有仓库部署镜像，您必须",
+  "create an image pull secret": "创建镜像 pull secret",
+  "with your image registry credentials.": "使用您的镜像 registry 凭证。",
+  "Secret {{newImageSecret}} was created.": "Secret {{newImageSecret}} 已创建。",
+  "Allow images from insecure registries": "允许来自不安全 registry 中的镜像",
+  "Image": "镜像",
+  "Deploy an existing image from an image stream or image registry.": "从镜像流或镜像 registry 中部署一个现有镜像。",
+  "No Image streams tags found": "未找到镜像流标签",
+  "No tags are available in image stream {{image}}": "镜像流中没有标签 {{image}}",
+  "No Image streams found": "未找到镜像流",
+  "No image streams are available in project {{namespace}}": "项目 {{namespace}} 中没有镜像流可用",
+  "Permission denied": "权限已拒绝",
+  "Service account default does not have authority to pull images from {{namespace}}. Select another project to continue.": "服务帐户默认值没有从 {{namespace}} 中拉取镜像的权限。选择另一个项目继续。",
+  "Grant service account default authority to pull images from {{namespace}}": "授予服务帐户默认权限从 {{namespace}} 中拉取镜像",
+  "No Image Stream": "没有镜像流",
+  "Select Image Stream": "选择镜像流",
+  "Image Stream": "镜像流",
+  "Project": "项目",
+  "Select Project": "选择项目",
+  "Tag": "Tag",
```

Comment:
"Tag" is translated into "tag" in ja/devconsole.json, and here is translated into "Tag", if it should be translated in Japanese and Chinese?

Reply:
This was not addressed.

Reply:
there's a devconsole_package=dev-console.po is this the package to look for? if so all instances of "tag" were already localized, I can send the file again just in case.

Reply:
Yes, that's the file! Thanks Terry.